schema/release-schema.json

```diff
@@ -68,7 +68,7 @@
     },
     "buyer": {
       "title": "Buyer",
-      "description": "The organization aiming to conclude a contract with a supplier or to use the goods, works or services resulting from the contract.",
+      "description": "The organization aiming to conclude a contract with a supplier or to use the goods, services and works resulting from the contract.",
```

Comment:
Was the conjunction changed from 'or' to 'and' deliberately?

Reply:
yes because it is possible for the buyer to be procuring a mixture of goods, services and works in a single contract

Reply:
fair enough, I'll change this back to 'or'